torchvision/prototype/transforms/functional/_deprecated.py

```diff
@@ -0,0 +1,71 @@
+import warnings
+from typing import Any
+
+import PIL.Image
+import torch
+
+from torchvision.prototype import features
+
+from ._meta import convert_color_space, convert_color_space_image_pil
+
+
+# TODO: this was copied from `torchvision.prototype.transforms._utils`. Given that this is not related to pytree / the
+#  Transform object, we should probably move it to `functional._utils`.
+def is_simple_tensor(inpt: Any) -> bool:
+    return isinstance(inpt, torch.Tensor) and not isinstance(inpt, features._Feature)
```

Comment:
Perhaps you are right that the original location at `torchvision.prototype.transforms._utils` might not be the best, but it's better than placing it here. Have you considered moving it to the `features` package? It's literally something that checks the type of the input. At any case, I wouldn't move this in this PR.

Reply:
> Have you considered moving it to the `features` package?

SGTM.